src/components/modules/paste.ts

```diff
@@ -254,14 +254,14 @@ export default class Paste extends Module {
    * Set onPaste callback handler
    */
   private setCallback(): void {
-    this.listeners.on(this.Editor.UI.nodes.holder, 'paste', this.handlePasteEvent);
+    this.listeners.on(document, 'paste', this.handlePasteEvent);
```

Comment:
Have you tested pasting on a page that has several editor instances? You can test on the `example-multiple.html` page

Reply:
I have checked earlier it was not working but I have added `event.stopImmediatePropagation() for the same not sure though, it will be good solution or not but it's working

Reply:
Fixed